### ISSUE TITLE
syntastic closes an open location list on Write

### ISSUE THREAD

**Issue by chrisbra** [scrooloose/syntastic#637]
Hi,
I just noticed, that synastic closes an already open location list. It would be nice, if when no error is found, syntastic would reopen the last open location list (e.g. by using :lolder or something a like).

Thanks.

**Comment by blueyed** [comment]
@lcd047 
Has the loclist_count branch been merged / used? (it does not exist anymore)
Is it https://github.com/scrooloose/syntastic/commits/loclist_state?

**Comment by lcd047** [comment]
@blueyed: Yes, it's controlled by `g:syntastic_reuse_loc_lists`.  But it's off by default (and undocumented) because of #1127.  I never got to the bottom of _that_ problem either.

**Comment by lorin** [comment]
Note that this issue breaks a vim-go feature:  https://github.com/fatih/vim-go/issues/814

**Comment by lcd047** [comment]
@lorin Please see #1650 instead.
